generated/nidaqmx/_library_interpreter.py

```diff
@@ -2268,7 +2268,7 @@ def get_analog_power_up_states_with_output_type(
             cfunc.argtypes = [
                 ctypes_byte_str, wrapped_ndpointer(dtype=numpy.float64,
                 flags=('C','W')), wrapped_ndpointer(dtype=numpy.int32,
-                flags=('C','W')), ctypes.c_uint, ctypes.c_uint32]
+                flags=('C','W')), ctypes.c_uint]
```

Comment:
The size is passed by pointer, so this should use `ctypes.POINTER` and `ctypes.byref` for the size. I think the original code may be wrong; it doesn't add the size to argtypes, so it will pass the size by value.
```
int32 __CFUNC_C DAQmxGetAnalogPowerUpStatesWithOutputType(const char channelNames[], float64 stateArray[], int32 channelTypeArray[], uInt32 *arraySizePtr);
```

https://github.com/ni/nidaqmx-python/blob/master/generated/nidaqmx/system/system.py#L633
```
        if cfunc.argtypes is None:
            with cfunc.arglock:
                if cfunc.argtypes is None:
                    cfunc.argtypes = [
                        ctypes_byte_str,
                        wrapped_ndpointer(dtype=numpy.float64,
                                          flags=('C','W')),
                        wrapped_ndpointer(dtype=numpy.int32,
                                          flags=('C','W'))]

        error_code = cfunc(
            flatten_channel_string(physical_channels), states, channel_types,
            size)
```